desktop/src/com/frostwire/gui/bittorrent/CreateTorrentDialog.java

```diff
@@ -99,6 +96,24 @@
     private String _invalidTrackerURL;
     private JButton _buttonClose;
 
+    private JComboBox _pieceSizeComboBox;
+    private JLabel _labelPieceSize;
+    private int pieceSize;
+    private static final String[] PIECE_SIZE_OPTIONS = {
+            "(Auto Detect)",
+            "16 kB",
+            "32 kB",
+            "64 kB",
+            "128 kB",
+            "256 kB",
+            "512 kB",
+            "1024 kB",
+            "2048 kB",
+            "4096 kB"
+    };
+
+    private static final int[] PIECE_SIZES = {0, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096};
```

Comment:
I did the test creating a dummy file and then creating a torrent from that file, piece sizes should by multiplied by 1024.